models/lorenz_96_tracer_advection/shell_scripts/run_filter_async4.csh

```diff
@@ -0,0 +1,227 @@
+#!/bin/csh
```

Comment:
we aren't supporting async4 anymore.  this file should be removed.

Reply:
removed https://github.com/NCAR/DART/tree/tracer_advection_siparcs